File: emulinker/src/main/java/org/emulinker/kaillera/model/event/InfoMessageEvent.java
package org.emulinker.kaillera.model.event;

import org.emulinker.kaillera.model.KailleraUser;

public class InfoMessageEvent implements UserEvent {
  private KailleraUser user;
  private String message;

  public InfoMessageEvent(KailleraUser user, String message) {
    this.user = user;
    this.message = message;
  }

  @Override
  public String toString() {
    return "InfoMessageEvent";
  }

  @Override
  public KailleraUser getUser() {
    return user;
  }

  public String getMessage() {
    return message;
  }
}

File: emulinker/src/main/java/org/emulinker/kaillera/model/event/InfoMessageEvent.kt
package org.emulinker.kaillera.model.event

import org.emulinker.kaillera.model.KailleraUser

class InfoMessageEvent(override val user: KailleraUser, val message: String) : UserEvent {
  override fun toString(): String {
    return "InfoMessageEvent"
  }
}
